supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/GetRequestHandler.scala

```diff
@@ -116,11 +116,11 @@ trait GetRequestHandler { this: RequestHandler =>
             case None => Success(executionEvents, machineType, zone, instanceName)
           }
         } else if (operation.hasStarted) {
-          Running
+          Running(executionEvents)
         } else {
-          Initializing
+          Initializing(executionEvents)
         }
-      catch {
+      } catch {
```

Comment:
Is there a reason we aren't creating the synthetic execution events for VM start/end time here? Are the events different for v2alpha1? We don't care about this backend?

Reply:
I will delete the v2alpha1 backend in another branch and merge it in here. Forgetting about genomics API ends today!